app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,17 @@
 import * as R from "ramda";
 
+const camelize = value =>
+  value
+    .toLowerCase()
+    .split(" ")
+    .reduce(
+      (word, character) =>
+        word + (character.charAt(0).toUpperCase() + character.slice(1))
```

Comment:
What if I give the value as `UX-Designer`? My expectation is that this util will emit `uxDesigner` even if there are not alphanumeric chars. Please verify and refactor if need be.

Reply:
Since I have used only a `whitespace` as a delimiter in the `split function`, the function will not give us the expected output. Updating the delimiter in the split function as `split(/[-_\s]/)`  that would handle whitespace, underscore and also dash(-), with this update upon giving `UX-Desginer` the function will now give the output as `uxDesigner` .
Shall I make this change ?

Reply:
You can the necessary change that will take into consideration the most commonly used non-alphanumeric chars into account.

Reply:
Updated the split function as `split(/[-_,./:\s]/)` taking into consideration the most common non-alphanumeric characters.